src/editor/deserialize.js

```diff
@@ -57,9 +57,10 @@ function parseHtmlMessage(html) {
 export function parseEvent(event) {
     const content = event.getContent();
     if (content.format === "org.matrix.custom.html") {
-        return parseHtmlMessage(content.formatted_body);
+        return parseHtmlMessage(content.formatted_body || "");
```

Comment:
Changes in this file seem unrelated to the rest of the PR, but I'll leave it up to you.

Reply:
Yeah, just some extra safety.